Update: Moved Alumni Card styles to App.css and updated component

Franckgou opened the pull request:
## WHY
The Alumni Card component currently has its styles in a separate CSS file. Management requested consolidation of CSS files to improve maintainability and reduce the number of separate style files in the project. Moving these styles to App.css will help centralize our styling approach.

## WHAT
- Moved all Alumni Card styles from separate CSS file to src/App.css
- Updated AlumniCard.html to reference App.css instead of its own CSS file
- Updated the CSS path in AlumniCard.html from "./AlumniCard.css" to "../App.css"
- Removed the separate AlumniCard.css file
- Maintained all existing styling and functionality

## TESTING
To test these changes:
1. Navigate to the Alumni Card component in the browser
2. Verify the glassmorphism effect is still working
3. Check that all styles are correctly applied:
   - Card background is semi-transparent
   - Profile image is circular
   - Text colors and spacing are maintained
   - Button styling is preserved
4. Verify responsive design on different screen sizes

kouedelo commented on src/App.css:
```diff
@@ -0,0 +1,99 @@
+  box-sizing: border-box;
+}
+
+body {
```
Can you please give another name to this body CSS class otherwise this styles will affect the body of the whole app.

Franckgou commented:
I think I will just erased the content of this file, I created it when I was making the Alumni card with Html and CSS, now since I'm using a jsx file it's no more useful to me

Get Outlook for Android<https://aka.ms/AAb9ysg>
________________________________
From: kouedelo ***@***.***>
Sent: Tuesday, December 17, 2024 10:02:52 PM
To: PkFokam-Alumni-Network/CommunityWebsite ***@***.***>
Cc: Franckgou ***@***.***>; Mention ***@***.***>
Subject: Re: [PkFokam-Alumni-Network/CommunityWebsite] Update: Moved Alumni Card styles to App.css and updated component (PR #28)


@kouedelo requested changes on this pull request.

________________________________

In src/App.css<https://github.com/PkFokam-Alumni-Network/CommunityWebsite/pull/28#discussion_r1889580599>:

> @@ -0,0 +1,99 @@
+/* Reset */
+* {
+  padding: 0;
+  margin: 0;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+
+body {


Can you please give another name to this body CSS class otherwise this styles will affect the body of the whole app.

—
Reply to this email directly, view it on GitHub<https://github.com/PkFokam-Alumni-Network/CommunityWebsite/pull/28#pullrequestreview-2510739397>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/BM6JJOXNREADPD4ZOBIBDI32GDXWZAVCNFSM6AAAAABTZFQSR6VHI2DSMVQWIX3LMV43YUDVNRWFEZLROVSXG5CSMV3GSZLXHMZDKMJQG4ZTSMZZG4>.
You are receiving this because you were mentioned.Message ID: ***@***.***>

kouedelo commented on src/App.css:
```diff
@@ -0,0 +1 @@
+/* Reset */
```
Please undo all changes in this file if there's nothing to be added.
